build.gradle

```diff
@@ -43,36 +54,219 @@ java {
     }
 }
 
-repositories {
-    // Use JCenter for resolving dependencies.
-    jcenter()
-    mavenLocal()
-    mavenCentral()
-    maven {
-        url 'https://jitpack.io'
+ext {
+    junitVersion           = '5.7.2'
+    junitPlatformVersion   = '1.7.2'
+    log4jVersion = '2.17.1'
```

Comment:
Should the log4jVersion property be set to 2.20.0 to match the version of log4j-slf4j-impl that's being used?

Reply:
Confirmed